www/templates/html/MediaPlayer/Video.tpl.php

```diff
@@ -28,8 +28,12 @@
     $autoplay = '';
 }
 
+$start_language = '';
+if (isset($controller->options['captions'])) {
+    $start_language = 'data-start-language="'.$controller->options['captions'].'"';
```

Comment:
Good point, it isn't being escaped at the moment. I will fix that.

Validation of this option at this time is not as important. If MediaElement is not able to validate the value, it will simply ignore it.